Version1.0

quyenhn opened the pull request:
First version

quyenhn committed:
Version1.0

ml-luanvvt commented on js/app.js:
```diff
@@ -0,0 +1,43 @@
+  {
+    if (prevImg.length!=0) prevImg.fadeIn().addClass('active');
+    else
+      $('img').last().fadeIn().addClass('active');
```
em xử lí duplicate code `.fadeIn().addClass('active')`

quyenhn replied:
function changeSlide(direct)
{
  var currImg = $('img.active');
  currImg.fadeOut().removeClass('active');
  if (direct == 'next') 
  {
    if (currImg.next('img').length)
      currImg = currImg.next('img');
    else
      currImg = $('img').first();
  } 
  else  
  {
    if (currImg.prev('img').length) 
      currImg = currImg.prev('img');
    else
      currImg = $('img').last();
  }
  currImg.fadeIn().addClass('active');
};

ml-luanvvt commented on slideshow.html:
```diff
@@ -0,0 +1,25 @@
+</head>
+<body>
+	<div class="slidebox">
+		<img src="images/1.jpg" class="active">
```
đoạn này nên bọc trong 1 element

ml-luanvvt commented on slideshow.html:
```diff
@@ -0,0 +1,25 @@
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+	<div class="slidebox">
```
xử lí để chạy 2 slideshow cùng 1 trang

quyenhn committed:
version2.0

-xử lý duplicate code
-bọc img va button trong 2 khối riêng biệt
-chạy 2slideshow trên cùng 1 trang ko ảnh hưởng lẫn nhau
